catalog/nsili/nsili-app/src/main/resources/features.xml

```diff
@@ -21,9 +21,15 @@
         <bundle>mvn:org.codice.alliance.nsili/catalog-nsili-source/${project.version}</bundle>
     </feature>
 
+    <feature name="catalog-nsili-endpoint" install="manual" version="${project.version}"
+             description="DDF Catalog STANAG 4559 Endpoint">
```

Comment:
Updated the description for both the source and the endpoint to reference Alliance NSILI vs STANAG 4559.